packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -43,8 +44,36 @@
   }
 }
 
+@mixin ratio-base() {
+  display: flex;
+
+  &::before {
+    content: '';
+    width: 1px;
+    margin-left: -1px;
+    float: left;
+    height: 0;
+    padding-top: aspect-ratio(4, 3);
+  }
+  &::after {
+    content: '';
+    display: table;
+    clear: both;
+  }
+}
+
 @mixin card-section {
   .#{$prefix}--card-section {
+    &__card {
+      .#{$prefix}--image {
+        @include ratio-base;
+
+        &__img {
+          height: auto;
+          flex: 1;
+        }
```

Comment:
I believe the aspect ratio is only on `@lg` breakpoint and higher. I think all other times it has a height of `240px`. Migth just want to double check with @larahanlon2 or someone.